types/node/ts3.7/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// NOTE: These definitions support NodeJS and TypeScript 3.7.
+
+// Base definitions for all NodeJS modules that are not specific to any version of TypeScript:
+// tslint:disable-next-line:no-bad-reference
+/// <reference path="../base.d.ts" />
```

Comment:
This needs to extend from the TS3.5 types.

```suggestion
/// <reference path="../ts3.5/index.d.ts" />
```